content/source/docs/extend/community/contributing.html.md

```diff
@@ -47,10 +52,10 @@ contributed.
 
 The current list of HashiCorp Providers is as follows:
 
- * `aws`
- * `azurerm`
- * `google`
- * `opc`
+ * [Amazon Web Services](https://github.com/terraform-providers/terraform-provider-aws)
+ * [Azure RM](https://github.com/terraform-providers/terraform-provider-azurerm)
+ * [Google Cloud Platform](https://github.com/terraform-providers/terraform-provider-google)
+ * [Oracle Public Cloud](https://github.com/terraform-providers/terraform-provider-opc)
```

Comment:
I assume the line here has become a bit more blurry with the redefined role of Ecosystem team, so the list may not be complete, but I reckon that's not the goal of this PR anyway.